### PR TITLE
Dev jack

### PR DESCRIPTION
**Pull request by Jack5hj**

### CODE REVIEW

File: lab-exercise-1/JackFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  public class JackFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (ChanAndKlee) [inline lab-exercise-1/JackFeature.java:3]: This function is incomplete

**Review by ChanAndKlee — COMMENTED** [review]
Please complete this by today

File: lab-exercise-1/JackFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+  public class JackFeature {
```
Comment (Jack5hj) [inline lab-exercise-1/JackFeature.java:1]: improve that

File: lab-exercise-1/JackFeature.java (hunk 1 above)
Comment (Pontuspts) [reply lab-exercise-1/JackFeature.java:3]: This content isn't well-written, please improve it.

Comment (NoodlesCupCake) [reply lab-exercise-1/JackFeature.java:3]: This content isn't well-written, please improve it.

File: JackFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+  public class JackFeature {
+      public void feature() {
+          // this is a new feature
```
[the remark this answers is not in the document]
Comment (NoodlesCupCake) [reply JackFeature.java]: This content isn't well-written, please improve it.

File: lab-exercise-1/JackFeature.java (hunk 2 above)
Comment (NoodlesCupCake) [reply lab-exercise-1/JackFeature.java:1]: This content isn't well-written, please improve it.

File: lab-exercise-1/JackFeature.java (hunk 1 above)
Comment (KRoos555) [reply lab-exercise-1/JackFeature.java:3]: This contents is not well-written. Please improve it.

**Review by NoodlesCupCake — COMMENTED** [review]
good now

File: JackFeature.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+public class JackFeature {
+      public void feature() {
+          // this is a new feature
+          int x = 0;
```
Comment (Pontuspts) [inline JackFeature.java:4]: good change

**Review by NoodlesCupCake — COMMENTED** [review]
good now

File: JackFeature.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+          x = x+1;
+      }
+  }
```
[the remark this answers is not in the document]
Comment (NoodlesCupCake) [reply JackFeature.java:7]: good again

Comment (NoodlesCupCake) [inline JackFeature.java:7]: good

**Review by Vvidsky — APPROVED** [review]

**Pull request merged**